### PR TITLE
Role: satellite-capsule-installation

### PR DESCRIPTION
**Pull request by miteshget**
Role: satellite-capsule-installation

Enhancement:
1: Firewall rules added
2: NTP tasks added
3: Default capsule repositories added
4: Readme file updated

### CODE REVIEW

File: ansible/roles/satellite-capsule-installation/tasks/firewalld.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ 
+- name: firewall | Enable and Start Firewalld
+  service:
+    name: firewalld
+    enabled: yes
+    state: started
+  tags:
+    - configure_capsule_firewalld
+
+
```
Comment (stencell) [inline ansible/roles/satellite-capsule-installation/tasks/firewalld.yml]: small nit: get rid of extra lines for consistency

File: ansible/roles/satellite-capsule-installation/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# - tftp
+# - libvirt-tls
+# - ldap
+# - ldaps
+# - docker-registry
+# - vnc-server
+
+# List of ports to add into the firewall via Firewalld
+# See README.md for an explanation.
+firewall_capsule_ports:
```
Comment (stencell) [inline ansible/roles/satellite-capsule-installation/defaults/main.yml]: small nit: if you don't need all these other ports, maybe cleanup for readability?

File: ansible/roles/satellite-capsule-installation/tasks/firewalld.yml (hunk 1 above)
Comment (miteshget) [reply ansible/roles/satellite-capsule-installation/tasks/firewalld.yml]: done

File: ansible/roles/satellite-capsule-installation/defaults/main.yml (hunk 2 above)
Comment (miteshget) [reply ansible/roles/satellite-capsule-installation/defaults/main.yml]: I have separated comments and used ports. Done

**Comment by miteshget** [comment]
@stencell , Please merge.

**Pull request merged**